composables/useCart.ts

```diff
@@ -89,6 +96,26 @@ export const useCart = () => {
     }
   }
 
+  const updateCartItem = async (cartItemId: string, cartItemInput) => {
```

Comment:
There should be an available Graphql type for cartItemInput

Reply:
Applied